src/utils/simulateNonSkillHelper.ts

```diff
@@ -0,0 +1,117 @@
+import * as vscode from 'vscode';
+import { Logger } from '../logger';
+import { loggableAskError } from '../exceptions';
+import { ERRORS, SIMULATOR_MESSAGE_TYPE } from '../constants';
+import { IViewport } from "apl-suggester";
+import { DEFAULT_VIEWPORT_CHARACTERISTICS } from "../aplContainer/utils/viewportProfileHelper";
+import { getCurrentDate } from '../utils/dateHelper';
+import { getSkillFolderInWs } from './workspaceHelper';
+import { read, write } from '../runtime/lib/utils/jsonUtility';
+import * as simulateMessageHelper from '../utils/simulateMessageHelper';
```

Comment:
I see this file is imported in `simulateMessageHelper` too. Won't this cause circular dependency problems now or in future? We should organize this better instead.

Reply:
Good point, is it better to import some methods but not the whole file?

Reply:
`simulateNonSkillHelper` uses two global variables in messageHelper, it should not use the methods in messageHelper. I will import only two variables instead :)